inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -228,16 +228,17 @@ def _prepare_update_payload(obj, eng):
 @with_debug_logging
 def refextract(obj, eng):
     uri = get_pdf_in_workflow(obj)
+    source = obj.data.get('acquisition_source', {}).get('source', None)
```

Comment:
You can write this as `get_value(obj.data, 'acquisition_source.source')`, and you get `get_value` from `inspirehep.utils.record`.